test/org/toml/lang/annotator/TomlAnnotatorTest.kt

```diff
@@ -0,0 +1,12 @@
+package org.toml.lang.annotator
+
+import org.toml.lang.TomlTestCase
+
+class TomlAnnotatorTest : TomlTestCase() {
```

Comment:
@matklad I've added the test
It would be nice to refactor this stuff to reuse ```Utils.kt``` in toml and rust test suites but I don't know how to organize packages in a right way, so I decided to leave it as is.